### PR TITLE
move background image into correct folder

### PR DESCRIPTION
**Pull request by liviu-preda-tsystems**
nmfluffy.jpg file is mixed into normal icons. As it is an icon protected under Telekom brand, and it is a core branding element, we should move it into img/telekom.

### CODE REVIEW

**Comment by tsdicloud** [comment]
Please leave small comment on PR to describe the purpose. THX!

File: css/apps/apps.scss
Comment (tsdicloud) [inline css/apps/apps.scss:1]: Are you sure you need to go up to server path?
I would assume `../../img/telekom/mcfluffy.jpg` should be enough?

**Review by tsdicloud — COMMENTED** [review]
Please shorten path if possible.

File: css/apps/apps.scss
Comment (liviu-preda-tsystems) [reply css/apps/apps.scss:1]: Done, seems to be working with `../../img/telekom/mcfluffy.jpg` now as well.

Comment (tsdicloud) [inline css/apps/apps.scss:1]: OK, got the point: as all styles are packed into nmctheme.css, `../img` is the right path. OK, sorry for missing that!

**Review by tsdicloud — APPROVED** [review]

**Pull request merged**